lib/modules/hover.js

```diff
@@ -413,22 +413,16 @@ addModule('hover', function(module, moduleID) {
 	HoverDropdownList.prototype = new Hover();
 	HoverDropdownList.prototype.templateName = 'RESHoverDropdownList';
 	HoverDropdownList.prototype._updatePosition = function($container) {
-		var XY = RESUtils.getXYpos(this._target);
-
-		var height = $(this._target).height();
-
-		var containerWidth = $container.outerWidth();
-
-		var left = XY.x;
-		if (left + containerWidth > window.innerWidth) {
-			left = window.innerWidth - containerWidth - 10;
-		}
-
-		$container.css({
-			top: XY.y + (height * 1.1) + 10,
-			left: left,
-			position: 'fixed',
-			zIndex: 2147483646
-		});
+		var trigger = $(this._target)[0];
+		var menu = $container[0];
```

Comment:
No need to avoid jQuery, I originally changed it to simplify it for myself, but we can def use jQuery there. I'll make a new PR with the cleaner code, after retesting.